SimPEG/EM/NSEM/SurveyNSEM.py

```diff
@@ -171,16 +195,16 @@ def fromRecArray(cls, recArray, srcType='primary'):
                 if np.any(notNaNind): # Make sure that there is any data to add.
                     locs = _rec_to_ndarr(dFreq[['x','y','z']][notNaNind].copy())
                     if dFreq[rxType].dtype.name in 'complex128':
-                        if 'z' in rxType:
-                            rxList.append(Point_impedance3D(locs,rxType[1:3],'real'))
-                            dataList.append(dFreq[rxType][notNaNind].real.copy())
-                            rxList.append(Point_impedance3D(locs,rxType[1:3],'imag'))
-                            dataList.append(dFreq[rxType][notNaNind].imag.copy())
-                        elif 't' in rxType:
+                        if 't' in rxType:
                             rxList.append(Point_tipper3D(locs,rxType[1:3],'real'))
                             dataList.append(dFreq[rxType][notNaNind].real.copy())
                             rxList.append(Point_tipper3D(locs,rxType[1:3],'imag'))
                             dataList.append(dFreq[rxType][notNaNind].imag.copy())
+                        elif 'z' in rxType:
```

Comment:
No, I don't. I can write some tests but this is bit of a legacy code which is being faced out since we can `pickle` the objects directly. 

How do best include files with tests?